src/compiler/axis.ts

```diff
@@ -0,0 +1,299 @@
+import {getter, setter} from '../util';
+import {Encoding} from '../Encoding';
+import * as util from '../util';
+import {X, Y, COL, ROW} from '../consts';
+import {Q, O, N, T} from '../consts';
+
+import * as time from './time';
+
+export default function(name: string, encoding: Encoding, layout, stats) {
```

Comment:
export default = replacing namespace?  

In any case, this should be `axis.def`